anvil-python/anvil/commands/haproxy.py

```diff
@@ -186,6 +187,28 @@ def get_unit_timeout(self) -> int:
         return HAPROXY_UNIT_TIMEOUT
 
 
+class UpgradeHAProxyUnitCharms(UpgradeCharm):
```

Comment:
Could you please upgrade all the other classes like this (maas-region, maas-agent, postgresql)?

```suggestion
class UpgradeHAProxyCharm(UpgradeCharm):
```